app/views/groups/index.html.erb

```diff
@@ -0,0 +1,21 @@
+<h1 class="text-center mt-4">All Catagories </h1>
+
+<div class="container">
+  <%@groups.each do |group|%>
+
+  <div class='card-contt'>
+    <div class='img-cont'>
+      <img src="<%= group.icon%>" class='img'>
+    </div>
+    <div class='cont-text'>
+      <h5><%=link_to group.name.capitalize , group_path(group), class:'underline text-color'%></h5>
+      <p>Total Amount :- <%= group.entities.sum(:amount) %> $</p>
+      <%= link_to 'View' , group_path(group), class:'bttn underline text-color'%>
+      <p class="text-color"> created <%= time_ago_in_words(group.created_at) %> ago</p>
+    </div>
+  </div>
+  <%end%>
```

Comment:
- [ ] Kindly make sure that when the user logs in, they are presented with the categories page, currently, they are redirected to the signout page, you can add the signout link as a link in your navigation bar.

Reply:
done